### ISSUE TITLE
VM Super slow without vmtools

### ISSUE THREAD

**Issue by assafnoahkoren** [hzqst/VmwareHardenedLoader#56]
I have a Windows machine setup according to the README guide.
It works and themida does not recognize its a VM!
But without vmtools the machine runs very slow and laggy.
Once I install vmtools the lags are gone but machine obviously gets recognized as a VM.
Any way to have the performance of a machine with vmtools but without actually installing vmtools?
Is it a drivers problem or something?

**Comment by ViktorVoloshko** [comment]
Nope. If you want to stay undetected you obviously need to avoid installing VMware stuff. Drivers as well.

**Comment by UltimateGG** [comment]
In the options without VMTools it says for 3d graphics acceleration you need vmtools. Could this be why it is so slow? It says VMTools installs a graphics driver maybe there is just a standalone one?
